src/frontend/src/pages/FinancePage/RefundsSection.tsx

```diff
@@ -0,0 +1,153 @@
+import {
+  AppBar,
+  Box,
+  LinearProgress,
+  Paper,
+  Stack,
+  Tab,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tabs,
+  Typography,
+  linearProgressClasses,
+  styled,
+  useTheme
+} from '@mui/material';
+import { useState } from 'react';
+import { useAllReimbursements, useCurrentUserReimbursements } from '../../hooks/finance.hooks';
+import ErrorPage from '../ErrorPage';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import { Reimbursement, ReimbursementRequest, User } from 'shared';
+import { useCurrentUser } from '../../hooks/users.hooks';
+import { fullNamePipe } from '../../utils/pipes';
+
+const NERProgressBar = styled(LinearProgress)(({ theme }) => ({
+  height: 20,
+  borderRadius: 5,
+  [`&.${linearProgressClasses.colorPrimary}`]: {
+    backgroundColor: theme.palette.grey[theme.palette.mode === 'light' ? 200 : 800]
+  },
+  [`& .${linearProgressClasses.bar}`]: {
+    borderRadius: 5,
+    backgroundColor: '#ef4345'
+  }
+}));
+
+const createRefundData = (date: Date, amount: number, recipient: User) => {
+  return { date, amount, recipient };
+};
+
+interface RefundTableProps {
+  currentUserRequests: ReimbursementRequest[];
+  allRequests?: ReimbursementRequest[];
+}
+
+const Refunds = ({ currentUserRequests, allRequests }: RefundTableProps) => {
+  const [value, setValue] = useState(0);
+  const user = useCurrentUser();
+
+  const { data, isLoading, isError, error } = useCurrentUserReimbursements();
+  const { data: allData, isLoading: allIsLoading, isError: allIsError, error: allError } = useAllReimbursements();
+  const theme = useTheme();
+
+  if (user.isFinance && allIsError) return <ErrorPage message={allError?.message} />;
+  if (isError) return <ErrorPage message={error?.message} />;
+  if ((user.isFinance && (allIsLoading || !allData)) || isLoading || !data) return <LoadingIndicator />;
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  const rows = (allData && value === 1 ? allData : data).map((row: Reimbursement) =>
+    createRefundData(row.dateCreated, row.amount, row.userSubmitted)
+  );
+  const totalReceived = (allData && value === 1 ? allData : data).reduce(
+    (accumulator: number, currentVal: Reimbursement) => accumulator + currentVal.amount,
+    0
+  );
+  const totalOwed = (allRequests && value === 1 ? allRequests : currentUserRequests).reduce(
+    (accumulator: number, currentVal: ReimbursementRequest) => accumulator + currentVal.totalCost,
+    0
+  );
+  const percentRefunded = (totalReceived / totalOwed) * 100;
+
+  return (
+    <Box sx={{ bgcolor: theme.palette.background.paper, width: '100%', borderRadius: '8px 8px 8px 8px', boxShadow: 1 }}>
+      <AppBar sx={{ borderRadius: '8px 8px 0 0' }} position="static">
+        <Tabs value={value} onChange={handleChange} indicatorColor="secondary" textColor="inherit" variant="fullWidth">
```

Comment:
I am still in very much so am in favor of making a tab container component and then passing in a { tabData:  { label: string, value: number }[], onChange: (value) => void } object to standardize the tabs

Reply:
I can do it for these two and see if it actually helps, this one would be separate from the ones on all the overview/detail pages though